src/AbstractRepository.php

```diff
@@ -52,4 +54,29 @@ protected function sanitizePath($path)
 
         return Path::canonicalize($path);
     }
+
+    /**
```

Comment:
I think you should add some PhpDoc with examples here that describes what exactly this method does.